fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/RoadSideUnitOperatingSystem.java

```diff
@@ -15,19 +15,16 @@
 
 package org.eclipse.mosaic.fed.application.app.api.os;
 
-import org.eclipse.mosaic.fed.application.ambassador.navigation.IRoutingModule;
+import org.eclipse.mosaic.fed.application.app.api.os.modules.AdHocCommunicative;
+import org.eclipse.mosaic.fed.application.app.api.os.modules.CellCommunicative;
+import org.eclipse.mosaic.fed.application.app.api.os.modules.Locatable;
+import org.eclipse.mosaic.fed.application.app.api.os.modules.Routable;
 
 /**
  * This interface extends the basic {@link OperatingSystem} and
- * is implemented by the {@link org.eclipse.mosaic.fed.application.ambassador.simulation.AbstractSimulationUnit}
- * {@link org.eclipse.mosaic.fed.application.ambassador.simulation.RoadSideUnit}.
+ * is implemented by the {@link org.eclipse.mosaic.fed.application.ambassador.simulation.RoadSideUnit}.
  */
-public interface RoadSideUnitOperatingSystem extends OperatingSystem {
+public interface RoadSideUnitOperatingSystem
```

Comment:
Just out of curiosity and because I'm to lazy to do the mental work:

We've talked about having stationary perception sensors before (e.g., in the form of RSUs or even an inheritor of RSU). What would be necessary apart from adding `Perceptive` here to make this work?

Reply:
Not so simple unfortunately. Currently the PerceptionModule needs specific data from the vehicle, such as projected position and heading. This data is currently obtained via `getVehicleData()` defined in the `PerceptionModuleOwner` interface. We certainly could refactor this, so that `RoadSideUnit` would return its position so that the `PerceptionModule` could use it. What needs discussion, and maybe even extension in mapping, is that we do not have a "heading" for RSUs yet, which the perception module would need in that case. We should better do that in a separate PR.